compiler/src/test/scala/org/scalajs/core/compiler/test/JSInteropTest.scala

```diff
@@ -198,25 +233,22 @@ class JSInteropTest extends DirectTest with TestHelpers {
   def onlyJSRawTraits: Unit = {
 
     """
-    @js.native
     trait A
```

Comment:
I was wondering for quite a while whether this was intended and what it should mean...